playbooks/roles/jenkins/tasks/upgrade_packages.yml

```diff
@@ -0,0 +1,6 @@
+---
+# Upgrades packages separately from the main Jenkins build playbook
+- name: Upgrade all packages to the latest version
```

Comment:
Might be worth rewriting to this to explicitly state that it is Jenkins requirements, rather than all installed packages?

Reply:
Well, everything on the box is in theory a Jenkins requirement 😸 These are ubuntu packages so I'd rather keep it generic.

Reply:
I will however change the comment to make it clear we're sticking to LTS versions.